include/drivers/circuits/hawickcircuits_driver.h

```diff
@@ -0,0 +1,69 @@
+/*PGR-GNU*****************************************************************
+File: circuits_driver.h
+
+Generated with Template by:
+Copyright (c) 2022 pgRouting developers
+Mail: project@pgrouting.org
+
+Function's developer:
+Copyright (c) 2022 Nitish Chauhan
+Mail: nitishchauhan0022@gmail.com
+
+------
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+ ********************************************************************PGR-GNU*/
+
+#ifndef INCLUDE_DRIVERS_CIRCUITS_HAWICKCIRCUITS_DRIVER_H_
+#define INCLUDE_DRIVERS_CIRCUITS_HAWICKCIRCUITS_DRIVER_H_
+
+
+/* for size-t */
+#ifdef __cplusplus
+#   include <cstddef>
+#else
+#   include <stddef.h>
+#endif
+
+typedef struct Edge_t Edge_t;
+typedef struct circuits_rt circuits_rt;
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************************************************
```

Comment:
This comment is also an inheritance of 2.0.
Please remove.
It is annoying that if you change the sql file, then you have to look for all the comments to fix this.
It is better not to have it.
In any case if you modify the SQL, the compiler will let you know what needs to be changed.

If you really need that reminder of how the SQL looks like, then while developing, you could have a link to:
https://github.com/pgRouting/GSoC-pgRouting/blob/nitish-2022/sql/circuits/_hawickCircuits.sql
but if you do, open an issue in your fork to remind you that this link has to be deleted before merging to pgRouting.